.editorconfig

```diff
@@ -13,9 +13,12 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 indent_style = tab
 
-[{package.json,*.yml}]
+[*.yml]
 indent_style = space
 indent_size = 2
 
+[*.md]
+trim_trailing_whitespace = false
+
 [{*.txt,wp-config-sample.php}]
```

Comment:
This theme repo will never have a `wp-config-sample.php` file in it. Plus the `*.txt` rule is very old and WordPress-specific for Windows people who open the WordPress readme file.

I suggest removing that part entirely.